e2e/__tests__/bees.test.js

```diff
@@ -0,0 +1,145 @@
+const request = require('../request');
+const { dropCollection } = require('../db');
+const User = require('../../lib/models/user');
+
+describe('BEEEES', () => {
+  beforeEach(() => dropCollection('users'));
+  beforeEach(() => dropCollection('bees'));
+
+  const testUser = {
+    email: 'me@me.com',
+    password: 'abc'
+  };
+
+  function makeFirstAdmin(admin) {
+    return request
+      .post('/api/auth/signup')
+      .send(admin)
+      .expect(200)
+      .then(({ body }) => body)
+      .then(user => {
+        return User.updateById(user._id, {
+          $addToSet: {
+            roles: 'admin'
+          }
+        });
+      })
+      .then(() => {
+        return request
+          .post('/api/auth/signin')
+          .send(admin)
+          .expect(200)
+          .then(res => {
+            admin = res.body;
+            return res.body;
+          });
+      });
+  }
+  let user = null;
+
+  beforeEach(() => {
+    return makeFirstAdmin(testUser).then(admin => (user = admin));
+  });
+
+  const bumbleBee = {
+    name: 'Bumble Bee',
+    appearance: {
+      pattern: 'stripe',
+      mainColor: 'yellow'
+    },
+    weight: 3,
+    conservation: 'endangered'
+  };
+
+  const editBee = {
+    name: 'Bumblebee',
+    appearance: {
+      pattern: 'stripe',
+      mainColor: 'yellow'
+    },
+    weight: 3,
+    conservation: 'endangered'
+  };
+
+  function postBee(bee) {
+    return request
+      .post('/api/bees')
+      .set('Authorization', user.token)
+      .send(bee)
+      .expect(200)
+      .then(({ body }) => body);
+  }
+  it('make a bee?', () => {
```

Comment:
you also want to be checking that a non-admin user cannot post, edit, or delete a bee